pkg/virtctl/ssh/ssh.go

```diff
@@ -185,3 +188,25 @@ func defaultUsername() string {
 	}
 	return ""
 }
+
+// Parse SSH Target argument supporting the form of username@vmi/name.namespace (or simpler)
+func Parse(arg string) (string, string, string, string, error) {
+	kind := "vmi"
+	var username, namespace, name string
+
+	usernameAndTarget := strings.Split(arg, "@")
+	if len(usernameAndTarget) > 1 {
+		username = usernameAndTarget[0]
+		if username == "" {
+			return "", "", "", "", errors.New("expected username before '@'")
+		}
+		arg = usernameAndTarget[1]
+	}
+
+	if arg == "" {
+		return "", "", "", "", errors.New("expected target after '@'")
+	}
+
+	kind, namespace, name, err := portforward.ParseTarget(arg)
+	return kind, namespace, name, username, err
```

Comment:
```suggestion
	return portforward.ParseTarget(arg)
```

Reply:
`portforward.ParseTarget(arg)` return only 4 variables and we need 5 variables

`portforward.ParseTarget(arg)`  does not return `username`

Reply:
My bad!